lib/Logger.jsx

```diff
@@ -21,57 +21,49 @@ export default class Logger {
 
     /**
     * Ask the server to write the log message
-    *
-    * @param {String} message The message to write
-    * @param {Number} recentMessages A number of recent messages to append as context
-    * @param {Object|String} parameters The parameters to send along with the message
     */
-    logToServer(message, recentMessages, parameters = {}) {
-        // Optionally append recent log lines as context
-        let msg = message;
-        if (recentMessages > 0) {
-            msg += ' | Context:  ';
-            msg += JSON.stringify(this.get(recentMessages));
-        }
-
-        // Output the message to the console too.
-        if (this.isDebug) {
-            this.client(`${msg} - ${JSON.stringify(parameters)}`);
+    logToServer() {
+        // We do not want to call the server with an empty list or if all the lines has onlyFlushWithOthers=false
+        if (!_.any(this.logLines, l => !l.onlyFlushWithOthers)) {
```

Comment:
I don't know why but I'm having trouble working out how `_.any` works here together with that comment. 
Can we maybe avoid negating `_.any` here? 

This kind of reads as return early if

>not any of the lines are not flagged with onlyFlushWithOthers

and the comment says to return early if... 

>if all the lines has onlyFlushWithOthers=false

Is that the same? I'm bad with double negatives..

Reply:
hmmm I can by changing it to:
```
if (!this.logLines.length || _.all(this.logLines, l => l.onlyFlushWithOthers)) {
```
It's a bit more verbose but does not have the double negations. It's also a tad less performant since it will check the whole array instead of exiting early when it finds one line that needs to be flushed, but I don't think it matters much here since at most it will have 50 entries.
What do you think, should I change it?

Reply:
Crap! I pushed this change with some other changes I had... LMK if you think the old version was better.

Reply:
I like this better!